playbooks/adhoc/uninstall.yml

```diff
@@ -55,6 +55,10 @@
     - pcsd
     failed_when: false
 
+  - shell: "atomic uninstall atomic-openshift-master"
+    changed_when: False
```

Comment:
@giuseppe That makes sense and, in sticking with the current convention, I don't think this needs to block this change but I do like seeing changed status when things have actually changed (if it's feasible to do).

I'm 99% sure the uninstall playbook is unsupported but perhaps we could follow up on this wrt the idempotentcy work that is being done. Uninstall won't be ran consecutively (no need to confirm that the playbook is doing unnecessary work on the N+1th run) so I'd consider this extremely low priority.